subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/configurations/DefaultConfigurationSpec.groovy

```diff
@@ -765,6 +764,35 @@ class DefaultConfigurationSpec extends Specification {
         assert copiedConfiguration.extendsFrom.empty
     }
 
+    def "Calls resolve actions and closures from original configuration when copied configuration is resolved"() {
+        Action<ResolvableDependencies> beforeResolveAction = Mock()
+        Action<ResolvableDependencies> afterResolveAction = Mock()
+        def config = conf("conf")
+        def beforeResolveCalled
+        def afterResolveCalled
+
+        given:
```

Comment:
Maybe extend the test case to demonstrate that a copy of a configuration with no before and after resolve action also has zero before and after resolve actions.